executor/executable/controllabletask.go

```diff
@@ -614,8 +614,8 @@ func (t *ControllableTask) Kill() error {
 				evt = "RESET"
 				destination = "STANDBY"
 			case "ERROR":
-				evt = "RECOVER"
-				destination = "STANDBY"
+				evt = "EXIT"
+				destination = "DONE"
```

Comment:
Shouldn't we still also allow the `RECOVER` transition anyway? I get it doesn't exist right now as an actual operation but IIRC the FSM accounts for it.

Reply:
yes, you're right

Reply:
No, but wait. We still do allow for it in `OccServer.cxx`, this has not changed. What has changed is that we EXIT instead of RECOVER when in ERROR in `Kill()`. Do you mean that we should try to RECOVER once before we EXIT? I do not see the point of that if we need to kill the tasks anyway.

Reply:
@knopers8 no, you're right, your change is correct. Thanks, merging.